src/prefect/runner/runner.py

```diff
@@ -423,15 +439,19 @@ async def stop(self):
                 "Exception encountered while shutting down", exc_info=True
             )
 
-    async def execute_flow_run(self, flow_run_id: UUID):
+    async def execute_flow_run(
+        self, flow_run_id: UUID, entrypoint: Optional[str] = None
+    ):
         """
         Executes a single flow run with the given ID.
 
         Execution will wait to monitor for cancellation requests. Exits once
         the flow run process has exited.
         """
         self.pause_on_shutdown = False
-        async with self:
+        context = self if not self.started else nullcontext()
```

Comment:
Hmm, I'm interested in the case where the runner hasn't started yet and we're trying to run autonomous-subflows on it (at least, that's how I'm interpreting this line)